sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -224,10 +224,12 @@
      */
     static final int QUERY_BATCH_LIMIT = 500;
 
-    @Nullable abstract String getProjectId();
-    @Nullable abstract Query getQuery();
-    @Nullable abstract String getNamespace();
-    abstract int getNumQuerySplits();
+    protected Read() {}
```

Comment:
Yeah, though this didn't work and it's still visible in the javadoc. And I can't make it private cause then AutoValue constructors won't work. Removed.